### DISCUSSION TITLE
Argument 1 passed to RateLimited::releaseAfterSeconds() must be of the type int (Laravel 8)

### DISCUSSION THREAD

**Discussion by whande1992** [spatie/laravel-rate-limited-job-middleware#36] [Q&A]
Hello, I'm trying to use the package in Laravel 8, but it's returning an error.

Argument 1 passed to Spatie\RateLimitedMiddleware\RateLimited::releaseAfterSeconds() must be of the type int,

```
 public function middleware()
    {

        $rateLimitedMiddleware = (new RateLimited())
            ->allow(1)
            ->everySeconds(1)
            ->releaseAfterOneMinute()
            ->releaseAfterBackoff($this->attempts(),3)
            ->key($this->instance->channel->plataform->name);


        return [$rateLimitedMiddleware];

      //  return [new RateLimited('limit_api')];
    }
```

Request limit is five per second maximum.
Laravel 8
PHP 7.4

LOG ERROR
https://pastebin.com/dL0wRxVR

JOB
https://pastebin.com/SjEH4rUm

**Comment by freekmurze** [comment]
The tests indicate that this is working as it should. Could you PR a failing test?
